OlinkAnalyze/vignettes/bridging_E3072toEHT.Rmd

```diff
@@ -0,0 +1,682 @@
+samples in the studies.
+
+### Normalization using the median of paired differences
+
+If it is expected that both the kind of distribution and the variance
+per test between runs are the same, then normalization using the median
+of paired differences will be preferred. Normalization using the median
+of paired differences based on the bridging samples is performed in the
+following steps:
+
+1.  For each assay in the Explore 3072 project, the pairwise difference
+    is calculated for each of the bridging samples with the Explore HT
+    project.
+
+2.  The normalization factor is estimated for each assay by finding the
+    median of the pairwise differences.
+
+3.  The assay-specific normalization factor for each assay is used to
+    normalize each data point from Explore 3072 to Explore HT.
+
+### Quantile smoothing
+
+Since Explore HT and Explore 3072 are two distinct products with
+different workflows involved in generating NPX data, some of the assays
+exist in corresponding but distinct NPX spaces. For those assays, the
+median of paired differences is insufficient for bridging as it only
+considers one anchor point (the median/50% quantile). Instead, quantile
+smoothing (QS) using multiple anchor points (5%, 10%, 25%, 50%, 75%, 90%
+and 95% quantiles) is favored to map the Explore 3072 data to the
+Explore HT distribution. The normalization using QS uses bridging samples
+to perform the following steps:
+
+1.  Each data point of the samples from Explore 3072 is mapped to the
+    equivalent space in Explore HT using an empirical cumulative
+    distribution function. An empirical cumulative distribution function
+    is a probability model which uses the observed data, in this case
+    the NPX values of the bridging samples for an assay, to create a step
+    function which interpolates linearly between the available data
+    points.
+
+2.  The empirical distribution function is used to map the data points
+    from Explore 3072 to the Explore HT space using the specified
+    quantiles. At this point all data points from the bridging samples
+    have NPX values that are normalized to the data points in Explore
+    HT.
+
+3.  To normalize the remaining data a spline regression model is
+    constructed using the sorted Explore 3072 data and the mapped
+    Explore 3072 data along with the anchor points of the spline
+    function. A spline regression model divides a data set at the
+    quantiles and uses the quantile as an anchor point or knot. Then a
+    model is generated to fit the points between each anchor point.
+
+4.  The spline regression model is then used to predict all the data
+    points from Explore 3072 to Explore HT. The spline regression model
+    results in a combination of linear regression models within
+    intervals. The Explore 3072 NPX values are input as the x value
+    within the corresponding interval, which results in a y value
+    equivalent to the Explore HT NPX value.
+
+### Function Output
+
```

Comment:
TBD - will we have the olink_normalization_bridge_product() wrapper ready for initial launch? If not, should we just refer to olink_normalization()? Will leave as-is for now.